app/controllers/bulk_downloads_controller.rb

```diff
@@ -75,6 +93,69 @@ def show
     render json: { error: BulkDownloadsHelper::BULK_DOWNLOAD_NOT_FOUND }, status: :not_found
   end
 
+  # GET /bulk_downloads/:id/presigned_output_url
+  def presigned_output_url
+    bulk_download_id = params[:id]
+
+    bulk_download = current_power.viewable_bulk_downloads.find(bulk_download_id)
```

Comment:
could you use `.find_by_id` here instead of `.find`? I'm wondering if someone could put strings in here like case statements

Reply:
Discussed offline.